### PR TITLE
Move sample-source to knative-sandbox

### PR DESCRIPTION
**Pull request by mattmoor**
Related: knative/community#153

/hold
/assign @chaodaiG @tcnghia

### CODE REVIEW

File: static/golang/sample-source.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<html><head>
+    <meta name="go-import" content="knative.dev/sample-source git https://github.com/knative-sandbox/sample-source">
```
Comment (chaodaiG) [inline static/golang/sample-source.html:2]: hmm. don't you need to delete the old record?

Comment (mattmoor) [reply static/golang/sample-source.html:2]: nobody ever created it.

Comment (chaodaiG) [reply static/golang/sample-source.html:2]: ack

Comment (chaodaiG) [reply static/golang/sample-source.html:2]: Actually, wondering how `dot-dev` worked for it before: https://github.com/knative/test-infra/pull/2244/files

**Pull request merged**